src/components/attendance/DataTableModal.tsx

```diff
@@ -58,6 +62,30 @@ const Body: React.FC<{ data: BodyProps | null | undefined }> = ({ data }) => {
   dayjs.extend(LocalizedFormat)
   const startDate = dayjs(data.startDate).format('lll')
   const endDate = dayjs(data.endDate).format('lll')
+  const modal = useContext(EventModalContext)
+  const confirmDelete = trpc.event.deleteEvent.useQuery(modal.id)
```

Comment:
We use useQuery when we are trying to retrieve things. In the case of a deletion, it is an example of a mutation as we are changing something, so here we will be using `useMutation` instead. You can refer to the other examples of useMutation existing in our code base and how it is written and also read up on some documentation.